server/api/settings.py

```diff
@@ -163,3 +163,14 @@
         'rest_framework_simplejwt.authentication.JWTAuthentication',
     )
 }
+
+# EMAIL
+
+EMAIL_BACKEND = 'django.core.mail.backends.smtp.EmailBackend'
+EMAIL_HOST = 'mailhog'
+EMAIL_PORT = 1025
+EMAIL_HOST_USER = ''
+EMAIL_HOST_PASSWORD = ''
+EMAIL_USE_TLS = False
+
+EMAIL_ADDRESS_FROM = 'email-tester@test.com'
```

Comment:
Store this in the .env.dev file when it goes to production we will have a different env file version. Probs need to store email_host and email_port etc in the env file as well because I assume they will be different for frontend

Reply:
I tried this, but I couldn't get it to work (emails stopped showing up in MailHog)

Reply:
You cant do  EMAIL_ADDRESS_FORM = os.environ.get("EMAIL_ADDRESS_FROM")?

Reply:
Thanks, I was trying that before and it wasn't working, but it's working now, so I thing I must have had the syntax subtly wrong